MapboxSearch/offline/src/main/java/com/mapbox/search/offline/OfflineSearchCallbackAdapter.kt

```diff
@@ -2,13 +2,19 @@ package com.mapbox.search.offline
 
 import com.mapbox.search.base.BaseResponseInfo
 import com.mapbox.search.base.BaseSearchCallback
+import com.mapbox.search.base.assertDebug
 import com.mapbox.search.base.result.BaseSearchResult
 
 internal class OfflineSearchCallbackAdapter(private val callback: OfflineSearchCallback) : BaseSearchCallback {
     override fun onResults(results: List<BaseSearchResult>, responseInfo: BaseResponseInfo) {
         callback.onResults(
-            results.map { OfflineSearchResult(it.rawSearchResult) },
-            OfflineResponseInfo(responseInfo.requestOptions.core.mapToOfflineSdkType())
+            results = results
+                .filter {
+                    assertDebug(it.name.isNotBlank()) { "Empty name" }
```

Comment:
As customers may be developing in debug mode, so let's remove this so that we do not interrupt their workflow. Instead, let's look at implementing something in our tests that can help identify similar issues.